src/app/(app)/specialist/[id]/page.js

```diff
@@ -22,9 +25,13 @@ export async function generateMetadata({ params }) {
 
 export default async function Page({ params }) {
   const { id } = params;
+  const cardStyle = 'mx-auto my-6 max-w-[900px] px-4 md:my-10 lg:px-0';
   const specialist = await getSpecialistById({ id });
+  const organization = !specialist && (await getOrganizationById({ id }));
 
-  return (
-    <CardSpecialist specialist={specialist} extended className="mx-auto my-6 max-w-[900px] px-4 md:my-10 lg:px-0" />
+  return specialist ? (
+    <CardSpecialist specialist={specialist} extended className={cardStyle} />
+  ) : (
+    <CardOrganization organization={organization} extended className={cardStyle} />
```

Comment:
@Yivgha it looks weird for me.

I would suggest to rewrite it like that

```js
const specialist = await getSpecialistById({ id });
if (specialist) {
   return <CardSpecialist specialist={specialist} extended className={cardStyle} />
 }
const organization = await getOrganizationById({ id });
return <CardOrganization organization={organization} extended className={cardStyle} />
```

Reply:
changed